web/islands/RemoteFollowModal.tsx

```
@@ -0,0 +1,332 @@
+                    </p>
+                    {actionInfo.value.software &&
+                      actionInfo.value.software !== "unknown" && (
+                      <p class="text-xs text-gray-500 dark:text-gray-500">
+                        {actionInfo.value.software.charAt(0).toUpperCase() +
+                          actionInfo.value.software.slice(1)}
+                      </p>
+                    )}
+                    {actionInfo.value.summary && (
+                      <p
+                        class="text-xs text-gray-600 dark:text-gray-400 mt-1 line-clamp-2 overflow-hidden"
+                        dangerouslySetInnerHTML={{
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)